monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -144,8 +145,28 @@ class ConfigurePageComponent extends AuthComponent {
       });
   }
 
-  updateMasqueStrings = () => {
-    console.log('MASQUE update');
+  updateMasqueStrings = async () => {
+    const [linuxRes, windowsRes] = await Promise.all([
+      IslandHttpClient.getBytes('/api/agent-binaries/linux/masque', {}, true),
+      IslandHttpClient.getBytes('/api/agent-binaries/windows/masque', {}, true)
+    ]);
+    const linuxMasqueBytes = await linuxRes.body.arrayBuffer();
+    const linuxMasqueStrings = this.getStringsFromBytes(linuxMasqueBytes);
+
+    const windowsMasqueBytes = await windowsRes.body.arrayBuffer();
+    const windowsMasqueStrings = this.getStringsFromBytes(windowsMasqueBytes);
```

Comment:
Do we really need all the async/await logic? Just use `then`?

Reply:
> It looks way more readable this way

I actually think it's more readable with `then`. :)
I'm also leaning towards it because that's how we have it almost everywhere else.

> @ordabach will want everything to use async/await anyway

Any specific reason?

Reply:
I like this way better. When using `then`, you tend to end up with things getting nested 10 layers deep. This is super clear. Call a function, get a return value, do the next thing.

Reply:
@mssalvatore already said it.